model/team.go

```diff
@@ -153,6 +153,10 @@ func (o *Team) IsValid() *AppError {
 		return NewAppError("Team.IsValid", "model.team.is_valid.description.app_error", nil, "id="+o.Id, http.StatusBadRequest)
 	}
 
+	if len(o.InviteId) == 0 {
```

Comment:
Should `InviteId` be removed from the `TeamPatch` struct?

Reply:
Good call, done.